cal/cal/models.py

```diff
@@ -650,6 +665,24 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         return events_qs.order_by('start')
 
+    def queryWeek(self, calendar_ids=None, start=None, end=None):
+        """
+        Returns a QuerySet of events matching this Tag.
```

Comment:
This does not actually return a `QuerySet`. Be careful about comments; good comment > no comment > bad comment